packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -220,7 +220,17 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
                 {profile?.name}
               </DescriptionListItem>
               <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
+                <div style={{display: 'flex', flexDirection: 'row'}}>
+                  <p>{profile?.themeColor}</p>
+                  <div
+                    style={{
+                      backgroundColor: profile?.themeColor,
+                      width: '30px',
+                      height: '20px',
+                      padding: '5px',
+                      marginLeft: '5px'
+                    }}></div>
+                </div>
               </DescriptionListItem>
```

Comment:
Is this way of adding css a good way in this case ?

Reply:
for a few css properties yes.